src/commands/php-errors.php

```diff
@@ -73,16 +73,43 @@ protected function execute( InputInterface $input, OutputInterface $output ) {
         $output->writeln( "<comment>Adding temporary bot collaborator to $site_domain.</comment>" );
 
         $add_collaborator_request = $api_helper->call_pressable_api(
-                "sites/{$pressable_site->id}/collaborators",
+                "collaborators/batch_create",
                 'POST',
                 array(
-                    'siteId' => $pressable_site->id,
-                    'email' => PRESSABLE_BOT_COLLABORATOR_EMAIL,
+                    'siteIds' => array(
+                        $pressable_site->id,
+                    ),
+                    'email'   => PRESSABLE_BOT_COLLABORATOR_EMAIL,
+                    'roles'   => 'sftp_access',
                 )
         );
 
-        if ( empty( $add_collaborator_request->data->accountId ) ) {
-            $output->writeln( "<error>Failed to create a temporary bot collaborator. Aborting!</error>" );
+        if ( ! is_null( $add_collaborator_request->errors ) ) {
+            $output->writeln( "<error>Error creating temporary bot collaborator. Aborting!</error>" );
+            exit;
+        }
+        // Get new collaborator id.
+        $max_tries = 0;
+        while ( empty( $collaborator_id ) || 3 >= $max_tries ) {
```

Comment:
The `$collaborator_id` variable doesn't exist yet. While `empty()` is designed to handle this, for readability purposes it's common practice to declare variables before using them:

```suggestion
        $collaborator_id = null;
        while ( empty( $collaborator_id ) || 3 >= $max_tries ) {
```

Also, yoda conditions for <, >, <= or >= are significantly more difficult to read and are best avoided: https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#yoda-conditions

```suggestion
        while ( empty( $collaborator_id ) || $max_tries <= 3 ) {
```